guides/parameters_binding.md

```diff
@@ -0,0 +1,52 @@
+# Parameters binding Guide
+
+This guide explains how the binding for request parameters and fields inside a parameter objects
+works and what happens when the binding fails.
+
```

Comment:
nit: one newline before and after headers is enough IMHO. This can lead to I like 3 newlines vs I like 2 newlines 😂